src/QsCompiler/Optimizations/Utils/Utils.fs

```diff
@@ -23,7 +23,7 @@ type internal InitKind = QsInitializerKind<ResolvedInitializer, TypedExpression>
 
 
 /// Represents the dictionary of all callables in the program
-type internal Callables (m: ImmutableDictionary<QsQualifiedName, QsCallable>) =
+type Callables (m: ImmutableDictionary<QsQualifiedName, QsCallable>) =
```

Comment:
This specific type has been eliminated with the latest updates I think.

Reply:
No, it's been used only within the Optimization project.

Reply:
Hmm, I would rather not introduce another public type Callables. While such an additional buffer makes sense for the optimization project alone, it's current functionality is rather limited and it is mostly there in anticipation of what might make sense in the future. I'd rather introduce an additional public class when it is actually needed. 
Would you hence mind just eliminating this class instead? Passing through ImmutableDictionary<QsQualifiedName, QsCallable> for now seems fine.

Reply:
Addressed in d964c6e